spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -728,6 +733,8 @@ public void testStructOfThreeLevelLists() throws Exception {
 
   @Test
   public void testTwoLevelList() throws IOException {
+    Assume.assumeTrue("Cannot migrate to a hadoop based catalog", !type.equals("hadoop"));
```

Comment:
why not use Assumptions from AssertJ?

Reply:
These two tests haven't been converted to AssertJ yet.